spec/acceptance/api/backend_api_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+resource 'BackendApi' do
+  let(:resource) { FactoryBot.create(:backend_api) }
```

Comment:
Do you need to create it? Does build work?

Reply:
@hallelujah Then it is going to save it anyway for this:
https://github.com/3scale/porta/blob/c9643b6dbbe16a44acf14ef318cfd591f5927e5b/spec/support/api/context.rb#L71

Reply:
And anyway I need the `id` and `created_at` which are only there for `create` and `build_stubbed` but i cannot do the later for that `save!`

Reply:
Yes that is why I think there is no need to create it and resave it :)

Reply:
Ok, I will put it as `build` then 😄 as long as we keep in mind that it is actually saving for that 😄